web/modules/custom/nys_registration/src/Form/RegisterForm.php

```diff
@@ -381,9 +336,9 @@ public function formBuildStep2(array &$form, FormStateInterface $form_state): ar
       $fid = $media->get('field_image')->target_id;
       $file = File::load($fid);
       $senator['image'] = empty($file) ?
-            '/themes/custom/nysenate_theme/src/assets/default-avatar.png' :
-            \Drupal::service('file_url_generator')
-              ->generateAbsoluteString($file->getFileUri());
+        '/themes/custom/nysenate_theme/src/assets/default-avatar.png' :
+        \Drupal::service('file_url_generator')
```

Comment:
Seems like you are injecting other services. Curious why you wouldn't for `file_url_generator`.

Reply:
Because I missed it.  Corrected.